docs/docs/identity-providers/readme.md

```diff
@@ -16,10 +16,18 @@ In this guide we'll cover how to do the following for each identity provider:
 
 1. Set a **[Redirect URL](https://www.oauth.com/oauth2-servers/redirect-uris/)** pointing back to Pomerium. For example, `https://${authenticate_service_url}/oauth2/callback`.
 2. Generate a **[Client ID]** and **[Client Secret]**.
-3. Configure Pomerium to use the **[Client ID]** and **[Client Secret]** keys.
+3. Generate a **[Service Account]** for additional IdP Data.
+4. Configure Pomerium to use the **[Client ID]**, **[Client Secret]**, and **[Service Account]** keys.
+
+:::warning
+
+You must configure an IdP **[Service Account]** to write policy against group membership, or any
+other metadata beyond username/e-mail.
```

Comment:
how about: "or any other data that is not directly returned from the id_token or UserInfo endpoint" ?

Reply:
> against group membership, or any other data that does not uniquely identify an end-user.

That covers any weird claim juggling we need to do to identify a user without implementation specifics.